src/HtmlParser.php

```diff
@@ -43,10 +43,12 @@ public function parseHtml( string $html ): DOMDocument {
 
 		$exception = null;
 		foreach ( $errors as $error ) {
-			if ( str_starts_with( $error->message, 'Tag template invalid' ) ) {
+			$msg = $error->message;
+			if ( str_starts_with( $msg, 'Tag ' ) && str_ends_with( $msg, " invalid\n" ) ) {
+				// discard "Tag xyz invalid" messages from libxml2 < 2.14.0(?)
```

Comment:
oof...

Reply:
Yeah, it’s annoying that it doesn’t happen for me locally, only in CI :S